src/components/Movie/Movie.css

```diff
@@ -18,10 +18,6 @@
   transform: scale(1.2);
 }
 
-.movie:focus {
-  padding: 2px;
-}
-
```

Comment:
Notes from discussion: removed because when tabbing the thicker fake border was being applied to all cards in the row, not just the one that was in focus.